packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -2140,6 +2140,7 @@
   "to_see_more_details_on_how_to_integrate": "to see more details on how to integrate.",
   "To_users": "To Users",
   "Toggle_original_translated": "Toggle original/translated",
+  "Toggle_link_preview": "Toggle link preview",
```

Comment:
Would it be possible to, instead of using the ambiguous `Toggle`, having the string change between `Enable` and `Disable` (or `enabled` and `disabled`)?

Reply:
I'm not sure, the string is used in this line: https://github.com/RocketChat/Rocket.Chat/pull/10755/files#diff-3c03382489c263d5149e9ce401f9f04fR315

If it is possible to add a string with modifiers (and its parameters) inside this `addButton()`, then sure, the problem is that even though I've seen it in other places, I have no idea of how to do that inside `addButton`, there are no other buttons that do this.

Another option would be to just have two buttons with opposing conditions, I don't like this one because it is a hack and inefficient.

Reply:
It seems that's what the ignore/unignore buttons do, add 2 buttons with opposing conditions. But it's still not a great way to do it...